docs/guides/sap/odata-connector.md

```diff
@@ -0,0 +1,170 @@
+  - [Destination Service](https://discovery-center.cloud.sap/serviceCatalog/destination?service_plan=lite&region=all&commercialModel=btpea), `lite` plan
+- One or more instance- or subaccount-level destinations, pointing to the SAP systems to communicate with.
+  ![sample BTP destination configuration](./img/btp-destination.png)
+- Ensure `Additional Properties` are correctly set on the Destination. For example:
+
+```json
+HTML5.DynamicDestination: true
+sap-client: <client/"Mandant" to work with on the SAP system>
+WebIDEEnabled: true
+WebIDESystem: <SAP system ID>
+WebIDEUsage: odata_gen
+```
+
+:::warning
+Currently, only `BasicAuthentication` is supported on the Destination by the SAP OData Connector.
+:::
+
+## Deployment to BTP
+
+A descriptor file is required to deploy the SAP OData Connector to a space in a SAP BTP subaccount. An exemplary deployment descriptor `mtad.yaml.example` is provided by Camunda. This is a standard format in SAP BTP's Cloud Foundry environment to describe the application that needs deployment. Take the following steps:
+
+1. Adjust the values to match those of the targeted Camunda 8 SaaS environment and rename it to `mtad.yaml`.
+2. Adjust the names of the SAP BTP destination and connectivity instances to your liking - both will be created automatically for you upon deployment. If instances in your subaccount of any of the two services exist, they will be reused.
+3. After creating the `mtad.yaml` file, log into the desired SAP BTP subaccount via the [Cloud Foundry `cli`](https://github.com/cloudfoundry/cli) (cf-cli):
+
+```shell
+$> cf login
+API endpoint: https://api.cf. ...
+...
+```
+
+4. Deploy the SAP OData Connector via the `cf-cli`. Note that this requires [the "multiapps" plugin of Cloud Foundry](https://github.com/cloudfoundry/multiapps-cli-plugin) to be installed on the machine the deployment runs on:
+
+```shell
+$> cf deploy ./ # append the -f flag to shortcircuit ongoing deployments
+Deploying multi-target app archive /Some/path/sap-odata-connector.mtar in org <your-org> / space <your-space> as you@example.org ..
+...
+Application "sap-odata-connector" started and available at "some.url.hana.ondemand.com"
+```
+
+## Deployment in Camunda 8 SaaS
+
+- If using Web Modeler, [import the SAP OData Connector's element template](/components/connectors/manage-connector-templates.md#importing-existing-connector-templates) for design use.
+
+![sample BPMN diagram with SAP OData connector](./img/sap-odata-connector-task-in-model.png)
+
+- If using Desktop Modeler, [follow the standard importing procedure](/components/modeler/desktop-modeler/element-templates/configuring-templates.md).
+
+## Working with the SAP OData Connector in Camunda Modeler
+
+### Modeling options
+
+Modeling the SAP OData Connector task is similar to any other Camunda Connector. When the BPMN task is selected, the properties panel in Modeler shows the respective options.
```

Comment:
Do we just mean selecting the wrench-shaped icon to change it to a type of Connector task?

Reply:
yes!